tests/particle_attributes/particle_attributes.xml

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<testproblem>
+  <name>Testing of particle attribute parameters in serial</name>
+  <owner userid="cmath"/>
+  <tags>flml</tags>
+  <problem_definition length="short" nprocs="1">
+    <command_line>        
+../../bin/fluidity -v3 -l particle-attributes.flml
+    </command_line>
+  </problem_definition>
+  <variables>
+    <variable name="attributes_error_1" language="python">
+import numpy as np
+from fluidity_tools import stat_parser
+s = stat_parser("particle_attributes.particles.Thomas")
+attributes_error_1 = np.zeros((1,60))
+
+for i in range(60):
+    name = 'Thomas_' + str(i+1).zfill(2)
+    attributes_error_1[0,i] = s[name]['Chem1'][0][-1] - 5.0
+print "max error",abs(attributes_error_1).max()
```

Comment:
Your print statements need brackets `()`. If abs(attributes_error_1).max() is the only thing you test for just make that the variable that's set here, rather than the entire array.